api/envoy/config/route/v3/route_components.proto

```diff
@@ -1659,6 +1659,7 @@ message RedirectAction {
       [(validate.rules).string = {well_known_regex: HTTP_HEADER_VALUE strict: false}];
 
   // The port value of the URL will be swapped with this value.
+  // If not specified, the request's port will be used.
```

Comment:
The "request port" is ambiguous as there can be multiple ports associated with a given request. Is it from `:authority`? from the actual L4 port?

Reply:
It is from headers, not L4.